server/command/command.go

```diff
@@ -236,7 +247,7 @@ func (c *command) deleteAlert(args ...string) (*model.CommandResponse, error) {
 	var message = "Successfully removed alert"
 	err := c.splunk.DeleteAlert(c.args.ChannelId, args[0])
 	if err != nil {
-		message = "Error while removing alert"
+		message = "Error while removing alert. " + err.Error()
```

Comment:
Log this error as well, if it's not being logged in the called function

Reply:
@mickmister sure , can you check on the message ,if you want some change